site/content/docs/5.2/forms/select.md

```diff
@@ -61,6 +61,13 @@ As is the `size` attribute:
 
 Add the `disabled` boolean attribute on a select to give it a grayed out appearance and remove pointer events.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This **disabled** `<select>` element should not be used because it does no respect the Orange Design System specifications.
```

Comment:
Don't understand it is perfectly valid to disable a select.

Reply:
What about:
> This **disabled** `<select>` element should not be used with this appearance because it does no respect the Orange Design System specifications.
> Please refer to the [Dropdowns](https://system.design.orange.com/0c1af118d/p/910b9b-dropdown/b/04c480) states guidelines on the Orange Design System website.

Reply:
Resolved ?